src/layouts/components/NavItems.tsx

```diff
@@ -63,7 +68,12 @@ export const NavItems = ({ setOpen }: NavItemsProps) => {
             ) : (
               <Button
                 component={Link}
-                onClick={() => handleClick(item.title, false)}
+                onClick={() => {
+                  item.dataCy === 'swapBase'
+                    ? setLifiWidgetOpen(true)
+                    : handleClick(item.title, false);
```

Comment:
I started to but since it's just an array that's being imported here I would need to create something new with state when there is already state being used in the NavItems via 
```
const [isLifiWidgetOpen, setLifiWidgetOpen] = useRootStore((state) => [
    state.isLifiWidgetOpen,
    state.setLifiWidgetOpen,
  ]);
```